### PR TITLE
command line tool

### PR DESCRIPTION
**Pull request by hernamesbarbara**
### Changes proposed in this pull request:

i made a little CLI for using num2words quickly from command line and thought i'd see if you thought this could be a useful addition to the project

### Status

- [ ] READY
- [ ] HOLD
- [X] WIP (Work-In-Progress)

### How to verify this change

* install via `pip` or `pip3` with my changes
* test command line app like this

```
$ num2words 24,120.10
twenty-four thousand, one hundred and twenty point one

# these are kinda fun if you're on a mac / have `say`
$ num2words 24,120.10 | say
$ num2words 24,120.10 -l es | say -v Jorge

```

### Additional notes

i like being able to convert large numbers into human readable format quickly

### CODE REVIEW

File: bin/num2words [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+def main():
+    version = "{}=={}".format(os.path.basename(__file__), __version__)
+    args = docopt(__doc__, argv=None, help=True, version=version, options_first=False)
+    if args["--list-languages"]:
+        for lang in get_languages():
+            sys.stdout.write(lang)
+            sys.stdout.write(os.linesep)
+        sys.exit(0)
+    try:
+        num = float(''.join(ch for ch in args['<number>'] if ch.isdigit() or ch == '.'))
```
Comment (shulcsm) [inline bin/num2words]: Why is it float? Also decimal and thousands separators are locale dependent. Imo just pass it down, and make the actual implementation do the parsing.

**Comment by erozqba** [comment]
@hernamesbarbara Thanks a lot for this contribution, I like the idea!
Could you please, look into the Travis-ci build and fix the problems?

**Comment by hernamesbarbara** [comment]
Hey, @erozqba. Sure! 

Full transparency, i haven't used travis or any CI tool before. But I _think_ the build problems are fixed now. Does it look OK to you? 

i'd love any tips for learning about travis :)

File: .coverage [hunk 2]
```diff
@@ -0,0 +1 @@
...2,43,44,45,47,48,49,50,51,52,53,54,56,57,59,60,61,62,64,65,67,68,70,71,72,74,75,76,78,79,81,82,84,85,87,88,90,91,93,94,96,97,98,100,101,102,104,105,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,135,136,138,139,140,141,142,144,145,147,148,149,151,152,153,154,155,157,158,160,161,163,164,166,167,169,170,171,173,174,175,177,178,180,181,182,183,184,185,186,187,188,190,191,192],"/Users/hernamesbarbara/code/github/hernamesbarbara/num2words/tests/test_uk.py":[16,18,20,23,24,59,62,76,81,60,63,64,66,67,69,70,72,73,25,26,28,29,30,31,33,34,35,36,86,87,88,90,91,92,94,95,96,104,105,106,107,109,110,111,112,114,115,116,117,124,125,126,127,129,130,131,132,78,79],"/Users/hernamesbarbara/code/github/hernamesbarbara/num2words/tests/test_utils.py":[1,3,6,7,8,9,10,11,13,15,16,17,19,20,22,23],"/Users/hernamesbarbara/code/github/hernamesbarbara/num2words/tests/test_vn.py":[17,19,21,24,26,29,35,43,52,61,79,99,107,27,62,63,64,65,67,68,69,70,72,73,74,75,76,53,54,55,56,58,59,36,37,38,39,40,41,30,31,32,33,44,45,46,47,48,49,50,80,81,82,84,85,86,88,89,90,91,93,94,95,100,101,103,104,111,112,114,115,116,117,120,121,122,123,124,128,129,132,133,135,136,137]}}
```
Comment (erozqba) [inline .coverage]: We don't need this file.

File: .travis.yml [hunk 3]
```diff
@@ -7,3 +7,4 @@ python:
   - "3.6"
 install: pip install tox-travis
 script: tox
+after_success: coveralls
```
Comment (erozqba) [inline .travis.yml]: We don't need this change, coveralls is called by tox.

File: bin/num2words (hunk 1 above)
Comment (erozqba) [reply bin/num2words]: I agree with @shulcsm here, let the actual implementation do the parsing.

**Comment by erozqba** [comment]
@hernamesbarbara I'm not an expert my self, but I have found that the documentation is really good. You should check it https://docs.travis-ci.com/.

**Comment by erozqba** [comment]
@hernamesbarbara do you have closed the pull request by error?
There are some PEP8 problems:
https://travis-ci.org/savoirfairelinux/num2words/jobs/330077015
https://travis-ci.org/savoirfairelinux/num2words/jobs/330077016
https://travis-ci.org/savoirfairelinux/num2words/jobs/330077017
https://travis-ci.org/savoirfairelinux/num2words/jobs/330077018

**Comment by hernamesbarbara** [comment]
ya i couldn't figure out how to make tox and/or travis work so i closed it

**Comment by erozqba** [comment]
Already merged on PR #193

**Pull request closed without merging**